source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -402,11 +402,33 @@ namespace AdaptiveSharedNamespace
 
     enum class ContainerBleedDirection
     {
-        BleedRestricted = 0,
-        BleedToLeading,
-        BleedToTrailing,
-        BleedToBothEdges,
+        BleedRestricted = 0x0000,
+        BleedLeft = 0x0001,
+        BleedRight = 0x0010,
+        BleedUp = 0x0100,
+        BleedDown = 0x1000,
+        BleedAll = BleedLeft | BleedRight | BleedUp | BleedDown
```

Comment:
>BleedAll = BleedLeft | BleedRight | BleedUp | BleedDown [](start = 8, length = 55)

this sadly doesn't work in android, it looks like java doesn't allow it operators between enums 

[https://stackoverflow.com/questions/6282619/is-there-a-way-to-bitwise-or-enums-in-java](https://stackoverflow.com/questions/6282619/is-there-a-way-to-bitwise-or-enums-in-java)